ceph.spec.in

```diff
@@ -57,6 +58,7 @@
 %bcond_with amqp_endpoint
 %bcond_with cephfs_java
 %bcond_with kafka_endpoint
+%bcond_with amqp_1_endpoint
```

Comment:
not sure whether this is supported or not in opensuse
so, for now, please remove this line

Reply:
i don't understand.
this line disables the feature for opensuse, unless you know that they don't support it, let leave the support for opensuse, and delete this line